vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -415,13 +415,14 @@ private Map getSecurityContext(Map config) {
 private void unstashWorkspace(config, prefix) {
     try {
         unstash "${prefix}-${config.uniqueId}"
-        echo "invalidate stash ${prefix}-${config.uniqueId}"
-        stash name: "${prefix}-${config.uniqueId}", excludes: '**/*', allowEmpty: true
     } catch (AbortException | IOException e) {
```

Comment:
Not strictly related to this PR, but why ignoring `IOException` in the first place and not raising this exception? This basically just hides the exception and can be really annoying to run down, right?